ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-nightly.yaml

```diff
@@ -1270,7 +1270,7 @@ tests:
     cluster_profile: azure-qe
     env:
       EXTRA_ARGS: --fips
-      HYPERSHIFT_HC_VERSION: "4.17"
+      HYPERSHIFT_CATALOG_IMAGE_VERSION: "4.16"
```

Comment:
The 4.17 job is using 4.16 image version, is that on purpose?

Reply:
Yup. The 4.17 catalog images will be available only after this minor version goes GA. Until then, OpenShift clusters are installed using catalog images of version x.(y-1).

Reply:
yes https://github.com/openshift/release/pull/53588#discussion_r1670750815